shared/src/business/useCases/generatePdfFromHtmlInteractor.js

```diff
@@ -31,19 +33,18 @@ exports.generatePdfFromHtmlInteractor = async (
 
     await page.setContent(contentHtml);
 
-    const headerContent = overwriteHeader
-      ? `${headerHtml || ''}`
-      : ` <div style="font-size: 8px; float: right;">
-              Page <span class="pageNumber"></span>
-              of <span class="totalPages"></span>
-            </div>
-            <div style="float: left">
-              ${headerHtml || `Docket Number: ${docketNumber}`}
-            </div>`;
+    if (headerHtml === undefined) {
```

Comment:
nit question: is this the same as `!headerHtml`? And if so, is one way of writing it better than the other?

Reply:
Really good question!  It is not the same as `!headerHtml`.  If `headerHtml` is an empty string (`''`), it is a falsy value.  That then means that `!headerHtml` would be be truthy and your custom header of nothing would be overwritten with the default header.  That's also why I added a new unit test to for this exact scenario.